### PR TITLE
Refs #26521 -- Accounted for mixins in CreateModel bases validation.

### PR DESCRIPTION
**Pull request by charettes**
Thanks Collin for the report.

### CODE REVIEW

File: django/db/migrations/operations/models.py [hunk 1]
```diff
@@ -56,12 +56,11 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
...
         # manager names
         _check_for_duplicates('fields', (name for name, _ in self.fields))
-        _check_for_duplicates(
-            'bases',
-            (base._meta.label_lower if isinstance(base, models.base.ModelBase) else base.lower()
-             for base in self.bases
-             if base is not models.Model)
-        )
+        _check_for_duplicates('bases', (
+            base._meta.label_lower if hasattr(base, '_meta') else
```
Comment (jiaaro) [inline django/db/migrations/operations/models.py]: what if two mixins have the same name?

hypothetically:  `facebook.models.UserMixin` and `twitter.models.UserMixin`

Comment (charettes) [inline django/db/migrations/operations/models.py]: Mixins are not converted to strings --they are kept as classes.

**Comment by collinanderson** [comment]
This fixes it for me. Thanks so much!

File: django/db/migrations/operations/models.py (hunk 1 above)
Comment (jiaaro) [inline django/db/migrations/operations/models.py]: oh, and only models are allowed to be named with strings - right you are 👍

File: tests/migrations/test_operations.py [hunk 2]
```diff
@@ -19,6 +19,10 @@
     sqlparse = None
 
 
+class Mixin(object):
```
Comment (timgraham) [inline tests/migrations/test_operations.py]: Doesn't work if it's inlined in the test method, correct? LGTM

Comment (charettes) [inline tests/migrations/test_operations.py]: That's what I originally tried doing but it resulted in a failure on Python 3 as the string representation of the class end up something like `<class 'migrations.test_operations.OperationTestBase.test_create_model_with_duplicate_base.<locals>.Mixin'>`

**Pull request merged**